File: app/src/main/java/com/example/demo/MainActivity.java
package com.example.demo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    Button btnlogin;
    EditText username , pwd;
    String uname = " ";
    String password = " ";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnlogin = findViewById(R.id.btn_submit);
        username = findViewById(R.id.et_username);
        pwd = findViewById(R.id.et_password);

        btnlogin.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {

                uname=username.getText().toString();
                password = pwd.getText().toString();

               if (uname.equals("tothenew") && password.equals("12345678"))
               {
                   Intent intent = new Intent(MainActivity.this, homePage.class);
                   Toast.makeText(getApplicationContext(),"login successful!!",Toast.LENGTH_SHORT).show();
                   startActivity(intent);
               }

               else
               {
                   Toast.makeText(getApplicationContext(), "Username or password is incorrect",Toast.LENGTH_SHORT).show();
               }
            }
        });

    }
}

File: app/src/main/java/com/example/demo/MainActivity.kt
package com.example.demo

import android.content.Context
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.PersistableBundle
import android.util.Log
import android.widget.Button
import androidx.fragment.app.DialogFragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentTransaction

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }



    override fun onStart() {
        super.onStart()
        Log.e("Activity-LIFE_CYCLE", "onStart")
    }

    override fun onResume() {
        super.onResume()
        Log.e("Activity-LIFE_CYCLE", "onResume")
    }

    override fun onPause() {
        super.onPause()
        Log.e("Activity-LIFE_CYCLE", "onPause")
    }

    override fun onStop() {
        super.onStop()
        Log.e("Activity-LIFE_CYCLE", "onStop")
    }

    override fun onDestroy() {
        super.onDestroy()
        Log.e("Activity-LIFE_CYCLE", "onDestroy")
    }
}